site/docs/pages/transaction/types.mdx

```diff
@@ -62,6 +67,16 @@ type TransactionReact = {
   className?: string; // An optional CSS class name for styling the component.
   contracts: ContractFunctionParameters[]; // An array of contract function parameters for the transaction.
   onError?: (e: TransactionError) => void; // An optional callback function that handles transaction errors.
+  onSuccess?: (response: TransactionResponse) => void; // An optional callback function that exposes transaction hash
+};
+```
+
+## `TransactionResponse`
+
+```ts
+type TransactionResponse = {
```

Comment:
Remember to export they types we document.

Reply:
this is in the docs though. we don't export here do we ?

Reply:
what I meant, is that when we document a type, we should also export it https://github.com/coinbase/onchainkit/blob/main/src/transaction/index.ts :)